src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -365,12 +355,32 @@ export default {
         openNotice() {
             this.notice = true;
         },
-
-        toggleOption(key) {
-            if (this.ballot.max_options === 1) {
-                this.votes = this.votes.includes(key) ? [key] : [];
+        shouldDisableCheckbox(key) {
+            return (
+                !this.isAuthenticated ||
+                !this.isBallotOpened(this.ballot) ||
+                (this.votes.length === this.ballot.max_options && !this.votes.includes(key))
+            );
+        },
+        displayBallotSelectionText() {
+            if (this.votes.length === 0) {
+                if (this.ballot.max_options > 1) {
```

Comment:
you can combine these conditionals, short circuiting on the length since no `else`